Hadrons/Modules/MScalar/ChargedProp.cpp

```diff
@@ -39,23 +40,18 @@ TChargedProp::TChargedProp(const std::string name)
 {}
 
 // dependencies/products ///////////////////////////////////////////////////////
-std::vector<std::string> TChargedProp::getInput(void)
+std::vector<std::string> TChargedProp::getInput()
 {
-    std::vector<std::string> in = {par().source, par().emField};
-    
-    return in;
+    return {par().source, par().emField};
```

Comment:
We should keep `void` in the argument signature and keep `in` as a separate `std::vector<std::string>` for consistency with the rest of Hadrons here.